LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIUtilities_LW.uc

```diff
@@ -358,126 +360,256 @@ function static string GetInfiltrationString(XComGameState_MissionSite MissionSt
 	return InfiltrationString;
 }
 
-function static BuildMissionInfoPanel(UIScreen ParentScreen, StateObjectReference MissionRef, bool IsInfiltrating)
+private static function GetMissionInfoPanelText(StateObjectReference MissionRef, bool IsInfiltrating, 
```

Comment:
Doesn't need to be private, and no side effects to removing it. I just designed GetMissionInfoPanelText to be called from BuildMissionInfoPanel in the same class.

Reply:
I meant more in the sense of whether the function itself has side effects, i.e. it modifies data outside of itself. It has out variables, so it kind of has side effects, but that's really a low-level detail of returning more than a single value. Plus all the `out` variables are primitives, so that's fine. Side effects would come from modifying a singleton game state object or something else defined outside of the function.

If a function has no such side effects, then I tend towards leaving it more open than closed. It's not a big deal, though.